### DISCUSSION TITLE
We can do that?

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#372] [Q&A]
![brave_km0JsnVBMi](https://github.com/Darkempire78/OpenCalc/assets/75314629/ea20869d-f54c-4c18-8184-66e3f626791b)

**Comment by clementwzk** [comment]
What is that ? Never saw that before

**Reply by inson1** [reply]
@Darkempire78 Its review site for apps - https://alternativeto.net/software/opencalc/about/

**Reply by clementwzk** [reply]
no we cannot do that

**Reply by inson1** [reply]
@Darkempire78 yea it sounded wierd xD I will try to get it deleted from there
